net/respondd/src/respondd.c

```diff
@@ -361,13 +417,132 @@ static void serve(int sock) {
 	}
 
 	if (output) {
-		if (sendto(sock, output, output_bytes, 0, (struct sockaddr *)&addr, addrlen) < 0)
+		if (sendto(sock, output, output_bytes, 0, addr, sizeof(struct sockaddr_in6)) < 0)
```

Comment:
I understand the idea, but i can't reproduce the compiler warning. Neither with clang nor with gcc.

```
$CC -ldl -ljson-c -std=c99 -D_GNU_SOURCE miniz.c respondd.c -Wall -Wextra -pedantic
```